contracts/AaveStrategy.sol

```diff
@@ -13,6 +13,7 @@ contract AaveStrategy is CommonContract {
     using SafeERC20 for IERC20;
     using Address for address;
     using SafeMath for uint;
+    using SafeMath for uint8;
```

Comment:
FYI solidity 0.8 comes with inbuilt safemath. you do not need to explicitly use any SafeMath library.